### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sethbergman**
I've fixed the formatting issues on the README.md file.

### CODE REVIEW

**Comment by ZHollingshead** [comment]
What was fixed precisely? Looking at the README.md file, the only thing I saw was that the `running the app` section was repeated.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,58 @@
...
+To run the app, execute the following command from within the app folder:
+
+```
+au run --env dev
+```
+
+See [the CLI documentation](https://github.com/aurelia/cli) for other available commands or type `au help` on the command line.
+
+```
+npm install
```
Comment (ZHollingshead) [inline README.md]: Information starting at line 33 is repeated here. I am guessing this is a mistake?

Comment (sethbergman) [reply README.md]: It's difficult to see with the diff, but the README.md has a `-` on just about every single line. There's nothing wrong with the information - just the formatting.

### Before:
aurelia-app-ux-showcase

### After:
sethbergman-app-ux-showcase-at-update-readme

Hopefully this helps clarify the issue. :smiley:

Comment (sethbergman) [reply README.md]: Ohh wait. Now I see what you're saying. Yes, let me get that fixed. :laughing:

**Pull request merged**